src/typechecker.erl

```diff
@@ -4435,11 +4439,6 @@ handle_type_error({type_error, record_pattern, Anno, Record, Ty}, Opts) ->
                Record,
                format_location(Anno, verbose, Opts),
                pp_type(Ty, Opts)]);
-handle_type_error({type_error, map, Anno, ExpectedTy}, Opts) ->
-    io:format("~sThe map~s does not have type ~s~n",
```

Comment:
Is map changed already, so this is dead code?

Reply:
Yes, I can't find any code that can throw `{type_error, map, _, _}`